psi-notify.service

```diff
@@ -1,9 +1,9 @@
 [Service]
+ExecStart=psi-notify
+ExecReload=kill -HUP $MAINPID
 Type=notify
-ExecStart=/usr/bin/psi-notify
-ExecReload=/bin/kill -HUP $MAINPID
+
 Restart=always
-Environment=DISPLAY=:0
```

Comment:
Hmm, I suppose that's fine. I was mostly wondering how people without `systemctl --user import-environment DISPLAY` would handle it, but I suppose they're already going to have that problem with other applications.